core/templates/dev/head/app.js

```diff
@@ -86,7 +87,7 @@ oppia.constant('EARLY_QUIT_THRESHOLD_IN_SECS', 45);
 oppia.constant('NUM_INCORRECT_ANSWERS_THRESHOLD', 5);
 oppia.constant('NUM_REPEATED_CYCLES_THRESHOLD', 3);
 oppia.constant('MAX_PLAYTHROUGHS_FOR_ISSUE', 5);
-oppia.constant('RECORD_PLAYTHROUGH_PROBABILITY', 0.2);
+oppia.constant('RECORD_PLAYTHROUGH_PROBABILITY', 1.0);
```

Comment:
Are the values of these constants what we expect to merge in? Or should we change them back to false?

Reply:
@brianrodri Ahhh. My bad, carried over from manual testing.